dist/images/ovnkube.sh

```diff
@@ -2400,7 +2452,10 @@ ovn-node() {
   fi
 
   echo "=============== ovn-node - (wait for ready_to_start_node)"
-  wait_for_event ready_to_start_node
+  if [[ ${ovnkube_node_mode} != "dpu-host" ]] && [[ ${ovn_enable_interconnect} != "true" ]]; then
+    echo "=============== ovn-node - (wait for ready_to_start_node)"
+    wait_for_event ready_to_start_node
+  fi
```

Comment:
we dont use ovnkube-node with IC right? I think we run it as part of ovnkube-controller combined now:
ovnkube-controller-with-node

Reply:
please add a smalls section here https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/docs/features/hardware-offload/dpu-support.md to show which parts of ovn-k run in which containers/pods on the node and DPU

Reply:
yeah if that is the only one you are actually testing, makes sense. Just make it clear that this is the only supported/tested setup

Reply:
added